Headers/DebugServer2/Host/Linux/ExtraWrappers.h

```diff
@@ -12,12 +12,38 @@
 #define __DebugServer2_Host_Linux_ExtraWrappers_h
 
 #include "DebugServer2/Base.h"
+#include "DebugServer2/Utils/CompilerSupport.h"
 
 #include <asm/unistd.h>
 #include <fcntl.h>
 #include <sys/syscall.h>
+#include <sys/user.h>
 #include <unistd.h>
 
+struct YMMHighVector {
+  uint8_t value[16];
+};
+
+struct xstate_hdr {
+  uint64_t mask;
+  uint64_t reserved1[2];
+  uint64_t reserved2[5];
+} DS2_ATTRIBUTE_PACKED;
+
+#if defined(ARCH_X86)
```

Comment:
This LGTM, can you just add a comment explaining why we need them (ptrace commands need these structures but they're not available in the system headers, etc).